lib/model.js

```diff
@@ -287,6 +287,10 @@ function operand (self, where, delta, data, val, op) {
   // disabled versioning?
   if (false === self.schema.options.versionKey) return;
 
+  // path excluded from versioning
+  var notVersionedPaths = self.schema.options.notVersionedPaths;
+  if (notVersionedPaths && notVersionedPaths.indexOf(data.path) >= 0) return;
```

Comment:
IMO `notVersionedPaths` should be stored as a map, no sense in storing it as an array if you only use it for lookups